driver/src/test/functional/com/mongodb/reactivestreams/client/Fixture.java

```diff
@@ -114,6 +122,49 @@ public static void drop(final MongoNamespace namespace) throws Throwable {
         }
     }
 
+    public static boolean serverVersionAtLeast(final int majorVersion, final int minorVersion) {
```

Comment:
Had to add these two methods.  Did it a bit differently that the Java driver since we don't have the underlying Cluster to get the description from.